glaciers.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Dec 11 08:15:21 2018
+
+@author: courtney
+"""
+
+import subprocess
+import os
+import matplotlib.pyplot as plt
+
+init_path = r"/home/courtney/Desktop/practice/"
+os.chdir(init_path)
+
+glaciers_path = r"/home/courtney/Desktop/practice/glaciers.txt"
```

Comment:
Same as line 12 - in general it's good practice to clearly comment (describe) any user-defined variables like paths that are necessary to run a script.

Reply:
Just saw this comment, ok thanks for the tips!!